### ISSUE TITLE
how to finetune the model on Posetrack dataset?

### ISSUE THREAD

**Issue by sklf** [Microsoft/human-pose-estimation.pytorch#80]
Hello, I am  trying to finetune your model on posetrack. I read your paper. You said the  learning rate strategy  was different and other settings were the same as those in the training COCO part.
I tried to finetune the model like what you said. But problem is COCO and Posetrack have 2 different keypoints -- "**head_top**" and "**head_bottom**" in posetrack, "**left_eye**", "**right_eye**" in COCO.
Before finetuning, I made a big COCO-format annotation file with those Posetrack json files. And I simply treated the "head_top" and "head_bottom" keypoints  as "left_eye", "right_eye". But it seemed like the model can't specify these 2 points very well.  
So can you tell me how you handle these 2 points?

**Comment by sklf** [comment]
I solved this problem. The reason is that I forgot to modify the self.flip_pairs. In COCO dataset, "3" means "left_eye" and "4" means "right_eye". They are in the flip_pairs. However, I made "3" corresponds with "head_top", and "4" corresponds with "head_bottom". The are **NOT**  in the flip_pairs.

**Comment by Odaimoko** [comment]
Hi, can you share the code? Have you trained on posetrack and how exactly? I have written code based on Posetrack17 format (with 15 joints), but cannont get the mAP in the paper.

**Comment by sklf** [comment]
> Hi, can you share the code? Have you trained on posetrack and how exactly? I have written code based on Posetrack17 format (with 15 joints), but cannont get the mAP in the paper.

I used the Posetrack18 dataset and generated a COCO-foramt json file. Follow the instruction in the paper, you should get the  mAP result presented. I get 77.4 mAP using Resnet50 model. But I'm trying to improve the MOTA performance. So I can't share code for now.
